### PR TITLE
REV-3: Adding custom testimonials

### PR DESCRIPTION
**Pull request by nmashton**
The hard-coded placeholder testimonials are replaced by (unfortunately hard-coded) real testimonials.

Images lifted from the Google Doc are added alongside appropriate text and captions.

The styles of testimonials on mobile devices are changed. The float-based layout was breaking in a bizarre way. This arranges the images in the desired "inline" fashion without similar breakage (although the result is still kinda funny-looking).

### CODE REVIEW

File: revolv/templates/base/home.html [hunk 1]
```diff
@@ -276,225 +276,43 @@
...
-          </a>
-        </li>
-        <li class="mobile-show">
-          <a href="javascript:;" class="head-img">
-            <i class="icons icon-arrow-down"></i>
-            <img src="{% static 'images/testimonial-img9.jpg' %}" class="desktop-head" alt="Head">
-            <img src="{% static 'images/mobile/mobile-testimonial-img9.jpg' %}" class="mobile-head" alt="Head">
-          </a>
-        </li>
+        {% with testimonial_image="images/testimonials/enio_ximenes.png" testimonial_image_mobile="images/testimonials/enio_ximenes.png" %}
```
Comment (trawick) [inline revolv/templates/base/home.html:279]: As these are added, please remove the old stock photos referenced in the previous version.

**Comment by trawick** [comment]
Can you try to debug the hammer.min.map thing?

File: revolv/templates/base/partials/testimonial-info-area.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<div class="info-area {% if not testimonial_active %}hide{% endif %}">
+  <p>
+    {{ testimonial_text }}
+  </p>
+  <div class="user-group">
+    <span class="block"><a href="javascript:;" class="link-blue">{{ testimonial_name }}</a></span>
+    <span class="txt block">{{ testimonial_position }}</span>
+    <p>
+      <emph>Have a project idea? Contact us at <a href="mailto:info@re-volv.org">info@re-volv.org</a>.</emph>
```
Comment (rlconley) [inline revolv/templates/base/partials/testimonial-info-area.html:9]: Is this "." supposed to be there </a>.</emph>?

Comment (kmtracey) [inline revolv/templates/base/partials/testimonial-info-area.html:9]: Yes, the dot at the end makes this a complete sentence, right?

Comment (rlconley) [inline revolv/templates/base/partials/testimonial-info-area.html:9]: Yes.

**Comment by rlconley** [comment]
:blue_heart:

**Pull request merged**